src/common/gui/SurgeBitmaps.h

```diff
@@ -13,4 +13,4 @@ class SurgeBitmaps
    void addEntry(int id);
 };
 
-VSTGUI::CBitmap* getSurgeBitmap(int id);
\ No newline at end of file
+VSTGUI::CBitmap* getSurgeBitmap(int id, bool newInstance = false);
```

Comment:
We should start documenting these starting from this commit. Lets us Doxygen: http://www.doxygen.nl/manual/docblocks.html. And I would recommend using the javadoc style e.g.

```
/**
 * @param id          <description>
 * @param newInstance <description>
 * @return description
 * Long description
 */
```

Lets not worry about generation at this point. Even having a structured style documented classes in the source code will help to understand.

Reply:
Yes, and this is an exception where using C-style comment is OK, because the tool requires it.

Reply:
Sure.